test/cmd/wait.sh

```diff
@@ -56,6 +54,43 @@ run_wait_tests() {
     kube::test::if_has_string "${output_message}" 'test-1 condition met'
     kube::test::if_has_string "${output_message}" 'test-2 condition met'
 
+    # create test data to test timeout error is occurred in correct time
+    kubectl apply -f - <<EOF
```

Comment:
This test is basically copied from this https://github.com/kubernetes/kubectl/issues/1219#issuecomment-1155501551